### PR TITLE
[Goldilocks] feat: Possible to disable argoproj rbac config

### PR DESCRIPTION
**Pull request by nissessenap**
**Why This PR?**
Give the possibility to remove argoproj.io from the clusterrole.

Fixes #
Give the possibility to disable unneeded clusterrole access

**Changes**
Changes proposed in this pull request:

* Added controller.rbac.enableArgoproj and dashboard.rbac.enableArgoproj to the helm chart

**Checklist:**

* [X] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Comment by lucasreed** [comment]
@NissesSenap This looks good to me, but please accept the CLA so that tests can run. Thanks!

**Comment by nissessenap** [comment]
Sorry for the delay @lucasreed , I have fixed the CLA now.

File: stable/goldilocks/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 appVersion: "v4.1.0"
-version: 5.1.0
+version: 5.1.1
```
Comment (sudermanjr) [inline stable/goldilocks/Chart.yaml]:
```suggestion
version: 5.2.0
```

**Comment by lucasreed** [comment]
@NissesSenap could you run `helm-docs --sort-values-order=file` at the root of the repo directory of your branch and commit that change. The helm-docs test is failing because the documentation doesn't match what that command outputs.

**Comment by nissessenap** [comment]
Which version of helm-docs are you using @lucasreed ?
I ran it locally and according to my version there is no changes happening to the file.

**Comment by lucasreed** [comment]
> Which version of helm-docs are you using @lucasreed ? I ran it locally and according to my version there is no changes happening to the file.

1.5.0

File: stable/goldilocks/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 appVersion: "v4.1.0"
-version: 5.1.2
+version: 5.3.0
```
Comment (sudermanjr) [inline stable/goldilocks/Chart.yaml]:
```suggestion
version: 5.2.0
```

**Review by sudermanjr — APPROVED** [review]

**Comment by nissessenap** [comment]
Sorry for the delay with the helm-docs thing. I had way to much to do lately, thanks for the help and the merge.

**Pull request merged**